en/upgrade-guide.md

```diff
@@ -24,33 +24,33 @@
                 // coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.3")
             }
     
-    * Unity Editor 버전에 따라 AGP 버전이 다르므로 올바른 버전을 확인하세요.
+    * The AGP version may vary depending on the Unity Editor version, so verify and use the appropriate version.
 
 ## 2.69.0
 
 ### Unity
 
-* GPGS AutoLogin를 사용하는 경우, **GetLastLoggedInProvider()** 동기 API 대신 신규 추가된 **RequestLastLoggedInProvider(GamebaseCallback.GamebaseDelegate\<string> callback)** 비동기 API를 사용하세요.
+* When using GPGS AutoLogin, use the newly added asynchronous API **RequestLastLoggedInProvider(GamebaseCallback.GamebaseDelegate<string> callback)** instead of the synchronous **GetLastLoggedInProvider()** API.
 
 ### Unreal
 
-* 약관 조회 결과 API인 FGamebaseQueryTermsResult가 수정되었습니다.
-    * TermsCountryType의 값이 설정되지 않는 문제를 수정했습니다.
-    * bPushEnabled, bAdAgreement, bAdAgreementNight가 제거되었습니다.
-* GPGS AutoLogin을 사용하는 경우, **GetLastLoggedInProvider()** 동기 API 대신 신규 추가된 **RequestLastLoggedInProvider(GamebaseCallback.GamebaseDelegate\<string> callback)** 비동기 API를 사용하세요.
+*  Fixed the terms query result API, FGamebaseQueryTermsResult.
+    * Fixed an issue where the value of TermsCountryType was not being set.
+    * bPushEnabled, bAdAgreement, bAdAgreementNight has been removed.
+* When using GPGS AutoLogin, use the newly added asynchronous API **RequestLastLoggedInProvider(GamebaseCallback.GamebaseDelegate<string> callback)** instead of the synchronous **GetLastLoggedInProvider()** API.
 
 ### Android
 
-* **gamebase-adapter-auth-gpgs-autologin** 모듈을 빌드에 포함하는 경우, **getLastLoggedInProvider()** 동기 API 대신 신규 추가된 **requestLastLoggedInProvider(GamebaseDataCallback&lt;String&gt;)** 비동기 API를 사용하세요.
+* When including the **gamebase-adapter-auth-gpgs-autologin** module in the build, use the newly added asynchronous API **requestLastLoggedInProvider(GamebaseDataCallback<String>)** instead of the synchronous **getLastLoggedInProvider()** API.
```

Comment:
HTML 기호를 특수문자로 변경하면 안됩니다. 마크다운 문서 렌더링시 표시가 안되게 됩니다.
```suggestion
* When including the **gamebase-adapter-auth-gpgs-autologin** module in the build, use the newly added asynchronous API **requestLastLoggedInProvider(GamebaseDataCallback&lt;String&gt;)** instead of the synchronous **getLastLoggedInProvider()** API.
```